src/main/kotlin/no/nav/amt/deltaker/bff/arrangor/ArrangorConsumer.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.amt.deltaker.bff.arrangor
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.nav.amt.deltaker.bff.application.plugins.objectMapper
+import org.slf4j.LoggerFactory
+import java.util.UUID
+
+fun consumeArrangor(id: UUID, arrangor: String?) {
+    val log = LoggerFactory.getLogger("ArrangorConsumer")
```

Comment:
Kanskje vi bør sette opp loggeren et sted så vi slipper å gjøre det hver gang metoden kalles?

Reply:
Jeg skal innrømme at jeg eksperimenterer litt her 😅 så jeg skal se hvordan det blir seende ut som en service også 👍  men her er tankene mine: 

Problemet med service er at det blir "mye" boilerplate til lite nytte her i mine øyne (loggeren caches så den opprettes bare en gang, og repositoriet har ingen state så det kunne like gjerne ha vært statisk).

Jeg har et alternativ til som jeg tror er funksjonelt ekvivalent med en service, hvor repositoriet er statisk så det kan mockes med `mockkObject`, og log deklareres på top nivå så man slipper å lete opp det hver gang.

Reply:
Jeg hater ikke servicen... 😅 

Det er også sikkert mer forståelig for andre slik, så det er jo positivt.

Reply:
Jeg reagerer nok litt instinktivt på det å newe opp ting på den måten inne i en metode, både med tanke på ytelse og testbarhet :) Hvis koden kan testes uten noe særlig styr og det ikke krever noe ekstra ressurser og objektene ryddes greit opp igjen så er det bare å eksperimentere i vei :D Skader ikke å gjøre ting på litt nye måter :)

Reply:
Etter å ha sovet litt på det tror jeg jeg lener mest mot servicen, det har litt andre fordeler hvor vi ikke trenger at repositoriet er en singleton hvis vi ikke vil det. Og jeg vil tro det er en mer stabil "oppskrift" å følge videre for de andre consumerne som kanskje kan trenge mer eller mindre avhengigheter 👍